### ISSUE TITLE
False positive with github actions yaml

### ISSUE THREAD

**Issue by dosas** [adrienverge/yamllint#666]
```
---
on:
  push:
    tags: ['**']
...
```

When running
```
 yamllint --strict 
```

Actual:
warning  truthy value should be one of [false, true]  (truthy)

Expected:
No warning

**Comment by dosas** [comment]
@adrienverge Could you be a little more verbose about it?

**Comment by andrewimeson** [comment]
@dosas this is an issue report with a lot of duplicates (e.g. #430). It's not a problem, it's intended default behavior.

Your options are:

1. Quote the key

   ```yaml
   ---
   name: Run tests

   "on": [push, pull_request]
   ```

2. Disable the rule with a line comment

   ```yaml
   ---
   name: Run tests

   on: [push, pull_request]  # yamllint disable-line rule:truthy
   ```

3. Disable key checking in the yamllint configuration (e.g. `.yamllint.yaml`)

   ```yaml
   ---
   extends: default
   rules:
     truthy:
       check-keys: false
   ```

**Comment by adrienverge** [comment]
Thanks @andrewimeson :pray:

Since https://github.com/adrienverge/yamllint/pull/650 and yamllint 1.34.0 there is yet another solution: declare the YAML version explicitly:
```yaml
%YAML 1.2
---
name: Run tests

on: [push, pull_request]
```
